manifests/repo.pp

```diff
@@ -33,6 +33,14 @@
         version => $version,
       }
     }
+    'Debian': {
+      if ! defined( '::apt' ) {
```

Comment:
yep it will fail. I wrote a comment about adding the module to the metadata.json and fixtures as well. This should ensure that the module is available.

---

There was a longer discussion about soft dependencies and hard dependencies, we decided for hard deps. So we should include the module here + in metadata